.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,55 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is. Please make an effort to fill in all the sections below; the information will help us investigate your issue.
+
+**Investigative information**
+- Durable Functions extension version:
+- Function App version (1.0 or 2.0):
+- Programming language used:
+
+***If JavaScript***
+- durable-functions npm module version:
+
+***If deployed to Azure***
+- Timeframe issue observed:
+- Function App name:
+- Function name(s):
+- Region:
+- Invocation IDs:
```

Comment:
Invocation ID isn't particularly useful. I would mark this as optional. What we _really_ want is the orchestration execution ID.

Reply:
Done; moved invocation ID field to 'additional info' section at bottom, changed 'orchestration instance ID' to 'orchestration execution IDs'.

Reply:
We actually want orchestration _instance_ IDs (which you had previously), not _execution_ IDs.

Reply:
I've opened #564 to address this.